File: IT/1/Funzioni/Esercizi/Java/Esercizio12.java
/* file Esercizio12.java */

public class Esercizio12 {
	private static int isum(int n) {
		int tot = 0;
		for (int i = 1; i <= n; i++) {
			tot += i;
		}

		return tot;
	}

	public static void main(String [] args) {
		System.out.println("Somma dei numeri da 1 a 100 = " + isum(100));
	}
}

File: IT/1/Funzioni/Esercizi/Python/esercizio12.py
# file esercizio12.py

from __future__ import print_function 

def isum(n):
	tot = 0
	for i in range(1, n+1):
		tot += i
	return tot

print("Somma dei numeri da 1 a 100 = {}".format(isum(100)))

